app/konnect/getting-started/configure-mesh.md

```diff
@@ -0,0 +1,103 @@
+1. Copy the personal access token that displays.
+1. From the left navigation menu in {{site.konnect_short_name}}, open [MESH MANAGER ICON HERE] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.
+1. Copy your control plane ID.
+1. Run the following to connect your global control plane to {{site.product_mesh_name}}:
+    ```sh
+    kumactl config control-planes add --address https://us.api.konghq.com/v0/mesh/control-planes/MGCP_ID/api --name mink-dev --headers 'authorization=Bearer KPAT'
+    ```
+    * **MGCP_ID:** Replace this with the control plane ID.
+    * **KPAT:** Replace this with the personal access token you created for the control plane.
+
+You can now run commands against your global control plane using `kumactl`. You can see the [`kumactl` command reference](/mesh/latest/generated/cmd/kumactl/kumactl/) for more information about the commands you can use.
+
+## Add services to your service mesh using the Kubernetes demo app
+
+Now that you've added a global control plane and a zone to your service mesh in {{site.konnect_short_name}}, you can add services to your mesh. You can use the {{site.mesh_product_name}} Kubernetes demo app to set up two services so you can see how {{site.mesh_product_name}} can be used to control services, monitor traffic, and track resource status.
+
+The demo application consists of two services:
+
+* `demo-app`: A web application that lets you increment a numeric counter
+* `redis`: A data store for the counter
+
+To add the services to your mesh using the demo app, run the following:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/kumahq/kuma-demo/master/kubernetes/kuma-demo-aio.yaml
+```
+
+You can see the services the Kubernetes demo app added by navigating to **Mesh Manager** in the sidebar of {{site.konnect_short_name}}, selecting the `example-cp` and clicking **Meshes** in the sidebar. You can view the services associated with that mesh by clicking **Default** and the **Services** tab.
+
+After you add a service to your service mesh, the {{site.konnect_short_name}} will be populated with metrics from your service mesh. For more information about the Kubernetes demo app, see [Explore {{site.mesh_product_name}} with the Kubernetes demo app](/mesh/latest/quickstart/kubernetes/).
+
+## Conclusion
+
+By following the instructions in this guide, you've created a {{site.mesh_product_name}} global control plane, added a zone to it, configured `kumactl` to connect to your global control plane, and added services to the control plane. 
+
+Now that you've completed this tutorial, you can continue to configure your service mesh in {{site.konnect_short_name}} by following some of these guides:
+
+* [Zone Ingress](/mesh/latest/production/cp-deployment/zone-ingress/) - Set up zone ingress in {{site.mesh_product_name}}.
+* [Zone Egress](/mesh/latest/production/cp-deployment/zoneegress/) - Set up zone egress in {{site.mesh_product_name}}.
```

Comment:
This will already be configured, we can delete this line.

Reply:
Done